ssz/src/main/java/tech/pegasys/teku/ssz/backing/cache/ArrayIntCache.java

```diff
@@ -40,52 +50,60 @@ private ArrayIntCache(V[] values, int initSize) {
     return (V[]) new Object[size];
   }
 
-  private void extend(int index) {
-    int newSize = values.length;
+  private V[] extend(int index) {
+    V[] valuesLoc = this.values;
```

Comment:
Should this be `valuesLock`?
```suggestion
    V[] valuesLock = this.values;
```

Reply:
I mean `Local` here :) Do you have a better naming?

Reply:
Hm... `valuesLocal` seems nice :)